### DISCUSSION TITLE
Ssid

### DISCUSSION THREAD

**Discussion by mrspeed1** [tbnobody/OpenDTU#1273] [Ideas]
Please accept ! And blanks in ssid which are default for avm Fritzboxes
Ahoy can handle this, just take the source from there. 
Thanks.

**Comment by Alf-Ele** [comment]
I don't think this is an openDTU issued.
My SSID is Fritz! Xxxxxxxxx, so with ! and blanks. And it works....

**Comment by mrspeed1** [comment]
You are right, i got it running.Tx.--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 26.08.23, 21:37 schrieb Alf-Ele ***@***.***>:

   
  I don't think this ist a openDTU issued. My SSID ist Fritz! Xxxxxxxxx, so with ! and blanks. And IT works....
